Update index.mdx (#1902)
OctopusDeploy/docs: src/pages/docs/infrastructure/workers/index.mdx

## Migrating to Workers \{#migrating-to-workers}

> Octopus workers provide a way to transition away from the built-in worker, enabling the shift from work being done on the Octopus Server to utilizing external workers, all without the need to update any deployment processes.  Learn about how to [use the default worker pool to move steps off the Octopus Server](/docs/infrastructure/workers/worker-pools/#Using-the-default-pool-to-stop-running-scripts-on-the-server).

Reviewer: How would you feel about this adjustment?

```suggestion
Octopus workers provide a way to move work off the built-in worker. This lets you move the work away from the Octopus Server and onto external workers without the need to update the deployment process. Learn [how to use the default worker pool to move steps off the Octopus Server](/docs/infrastructure/workers/worker-pools/#Using-the-default-pool-to-stop-running-scripts-on-the-server).
```
